File: test/corpus/languages/java/fixture/src/test/java/com/example/MainTest.java
package com.example;

public class MainTest {
    public void testCreateSampleUser() {
        User user = Main.createSampleUser();
        assert user.getName().equals("John");
    }

    public void testUserRepository() {
        Storage storage = new MemoryStorage();
        UserRepository repo = new UserRepository(storage);
        User user = Main.createSampleUser();
        repo.addUser(user);
    }
}

File: test/corpus/languages/php/fixture/tests/MainTest.php
<?php

namespace LetaFixture\Tests;

use LetaFixture\Main;
use LetaFixture\MemoryStorage;
use LetaFixture\UserRepository;

class MainTest {
    public function testCreateSampleUser(): void {
        $user = Main::createSampleUser();
    }

    public function testUserRepository(): void {
        $storage = new MemoryStorage();
        $repo = new UserRepository($storage);
        $user = Main::createSampleUser();
        $repo->addUser($user);
    }
}
